pages/linux/sslstrip.md

```diff
@@ -0,0 +1,24 @@
+# sslstrip
+
+> Perform Moxie Marlinspike's Secure Sockets Layer (SSL) stripping attacks. Perform an ARP spoofing before using this tool.
+> More information: <https://www.kali.org/tools/sslstrip/>.
+
+- Listen on a specific port:
+
+`sslstrip --listen={{target_port}}`
+
+- Log all SSL traffic to and from the server:
```

Comment:
Maybe we can keep all of the descriptions consistent, the only variation from the one above is the --ssl flag.

So maybe

```suggestion
> Listen to HTTPS trafic on a specific port:
```

And apply this sort of change throughout.


Also it isn't currently logging really, since its not being wrote anywhere (from what I gather)

Reply:
The description is aligned with the kali document. IMO, it is better to use the document description rather than create our own description.

If the --write is not specified, the sslstrip will log it to a default file: sslstrip.log.